hazelcast/src/test/java/com/hazelcast/config/ConfigXmlGeneratorTest.java

```diff
@@ -383,6 +382,33 @@ public void testTlsHostVerification() {
 
     }
 
+    @Test
+    public void testScheduledExecutor() {
```

Comment:
Can we rename this test method to reflect exact testing goal? Now i need to read the whole test to understand what it is testing.